pkg/audit/cursor.go

```diff
@@ -18,78 +18,78 @@ import (
 	"storj.io/storj/pkg/storage/meta"
 )
 
-// Audit to audit segments
-type Audit struct {
+// Stripe keeps track of a stripe's index and its parent pointer
+type Stripe struct {
+	Index   int
+	Pointer *pb.Pointer
```

Comment:
I would call this field `Segment`. `Stripe.Segment.Size` looks more natural than `Stripe.Pointer.Size`.

Reply:
I don't think Stripe.Pointer.Size is called anywhere, but I'll still change it.

Reply:
I meant it just as an example for accessing this field.

In the end the caller of `NextStripe` will access `Stripe.Segment` and its fields to attempt to download the stripe from the storage nodes.

Reply:
That's true. But the parts of the code that do this refer to the pointer as a `pointer`, e.g. this part from DownloadShares:
```
	shareSize := int(pointer.Remote.Redundancy.GetErasureShareSize())
	pieceID := client.PieceID(pointer.Remote.GetPieceId())

	for i, node := range nodes {
		paddedSize := calcPadded(pointer.GetSize(), shareSize)
		pieceSize := paddedSize / int64(pointer.Remote.Redundancy.GetMinReq())
...
```
In this instance should I change `pointer` to `segment` too?

Reply:
If it reads better then it makes sense.